### PR TITLE
Class

### PR DESCRIPTION
**Pull request by moshesrur**
Exercise 1- Person class
Exercise 2- Summer class

### CODE REVIEW

File: class-lab/exercise3/car.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  isFasterThan(other) {
+    return this.speed > other.speed;
+  }
+}
+
+class Race {
+  cars = [];
+
+  constructor() {
+  }
```
Comment (ilyador) [inline class-lab/exercise3/car.js]: No need to keep unused code.

File: class-lab/exercise3/car.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+  addCars(c1, c2, c3) {
+    this.cars[0] = c1;
+    this.cars[1] = c2;
+    this.cars[2] = c3;
+  }
```
Comment (ilyador) [inline class-lab/exercise3/car.js]: If you already have learned about the spread operator, you can define a function that has an unlimited number of arguments, like this:
```js
addCars(...cars) {
  this.cars = [...cars]
}
```

File: class-lab/exercise1/person.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+class Person {
+  constructor(name, age=1) {
+    this.name = name;
+    this.age = age;
+  }
+
+  hello() {
+    console.log("My name is "+ this.name + " and I am " + this.age + " years old");
```
Comment (ilyador) [inline class-lab/exercise1/person.js]: To keep code a little cleaner you can use ES6 template strings, like this:
```js
console.log(`My name is ${this.name} and I am ${this.age} years old`)
```

**Review by ilyador — COMMENTED** [review]
Your code is great, I just added some minor improvements that could be done, but nothing critical.

File: class-lab/exercise3/car.js (hunk 1 above)
Comment (moshesrur) [reply class-lab/exercise3/car.js]: removed

File: class-lab/exercise3/car.js (hunk 2 above)
Comment (moshesrur) [reply class-lab/exercise3/car.js]: changes, Thanks!

**Pull request merged**